packages/modelserver-client/src/model-server-client-api-v1.ts

```diff
@@ -28,7 +30,7 @@ export interface ModelServerClientApiV1 {
      * @param defaultFormat Optional fallback format that should used when a request method is invoked and no explicit format argument
      *                      has been passed.
      */
-    initialize(baseUrl: string, defaultFormat?: string): void | Promise<void>;
+    initialize(baseUrl: URI, defaultFormat?: string): void | Promise<void>;
```

Comment:
I'm not sure about using the `URI` type for the HTTP resource/request URL. It's not really exactly an URI. Perhaps instead this should be the Node built-in [`URL`](https://nodejs.org/api/url.html) type. What do you think?

Reply:
I agree, the baseUrl is actually an URL (identifying the endpoint **location** as our jsdoc states) and not an URI, so using the URL type makes sense here.

Reply:
I would like to stick to one type, that makes things a little more unified. Also the documentation of the URI library speaks of URLs being a "subset" of the URIs according to definition: https://medialize.github.io/URI.js/about-uris.html

Reply:
Yes URLs are a subset. In the end in doesn't matter. I agree that for code simplification it make sense to stick to one type.